src/components/UploadIdsModal.js

```diff
@@ -2,16 +2,17 @@ import React from 'react';
 import { compose } from 'recompose';
 import { Trans } from 'react-i18next';
 import { withTheme } from 'emotion-theming';
-import { css } from 'react-emotion';
 import { injectState } from 'freactal';
 
 import { MatchBox } from '@arranger/components/dist/Arranger';
 import graphql from 'services/arranger';
 import { ModalFooter } from './Modal';
+import { TealActionButton } from 'uikit/Button';
+import { Paragraph } from 'uikit/Core';
+import { FileRepoH3 as H3 } from 'uikit/Headings';
+import { TableHeader } from 'uikit/Table';
 
-const UploadButton = withTheme(({ theme, ...props }) => (
-  <button className={theme.actionButton} {...props} />
-));
+const UploadButton = withTheme(({ theme, ...props }) => <TealActionButton {...props} />);
```

Comment:
this component can go, it's not exported so no risk of breaking stuff. `TealActionButton` can be consumed directly now